server/views/layout/_preview.njk

```diff
@@ -5,9 +5,13 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <title>Wellcome Collection</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+  <style>
+	 {% include "views/partials/css-head.njk" %}
+  </style>
   <link rel="stylesheet" href="{{ '/assets/css/styleguide.css' | path }}">
-  {# TODO Temporary until decide on font hosting solution #}
-  <script type="text/javascript" src="//fast.fonts.net/jsapi/b026248c-28d7-40e1-b542-97c76e7a74fd.js"></script>
+  <script>
+	 {% include "views/partials/js-head.njk" %}
```

Comment:
This seems like a bit of a hack and were committing compiled code here.
I'm not 100% sure what the solution is - but could we at least have a `TODO` to get rid of this later?